### PR TITLE
Remove old NPM code generator

### PR DESCRIPTION
**Pull request by sav007**
- clean up gradle plugin
- refactor gradle plugin tests by removing anything related to Node tasks
- remove reference to old apollo codegen npm module
- clean up graphql compiler tests

### CODE REVIEW

**Comment by sav007** [comment]
cc @martinbonnin

File: apollo-compiler/src/test/kotlin/com/apollographql/apollo/compiler/CodegenTest.kt [hunk 1]
```diff
@@ -114,6 +106,8 @@ class CodeGenTest(val pkgName: String, val args: GraphQLCompiler.Arguments) {
       return File("src/test/graphql/com/example/")
           .listFiles()
           .filter { it.isDirectory }
+          // TODO figure out what to do with these cases
+          .filter { it.name != "nested_inline_fragment" && it.name != "reserved_words" && it.name != "scalar_types" }
```
Comment (martinbonnin) [inline apollo-compiler/src/test/kotlin/com/apollographql/apollo/compiler/CodegenTest.kt]: Is this something we don't support with the new antlr IRGen ?

**Comment by martinbonnin** [comment]
Awesome! I was going to ask when we could remove this code.

File: apollo-compiler/src/test/kotlin/com/apollographql/apollo/compiler/CodegenTest.kt (hunk 1 above)
Comment (sav007) [reply apollo-compiler/src/test/kotlin/com/apollographql/apollo/compiler/CodegenTest.kt]: well the issue with these test fixtures that they were generated manually, there are no corresponding GraphQL queries. IR.json files for these test were written by hand. With new code gen we need to modify schema itself, it's very tedious task. I will try to do this in next PR.

**Pull request merged**